### PR TITLE
Lurebox

### PR DESCRIPTION
**Pull request by adnanavni**
lurebox added to profile and made a path to /newbait where you can add new baits and add baits to your lurebox

closes https://github.com/perttuvaarala/CatchMaster/issues/8

### CODE REVIEW

File: packages/app/src/components/Posts.tsx [hunk 1]
```diff
@@ -54,19 +53,13 @@ const Post: FC<PostProps> = ({ post }) => {
...
 	);
 };
 
 export default function Posts() {
 	const { loading, error, data } = useAllPostsQuery();
 
-	/*const [createPostMutation] = useCreatePostMutation();
-  const handleCreatePost = () => {
-    createPostMutation({variables:{}})
-  }*/
```
Comment (pennane) [inline packages/app/src/components/Posts.tsx:68]: Should this be removed? Is perttu still working on this?

File: packages/app/src/components/Posts.tsx [hunk 2]
```diff
@@ -76,7 +69,11 @@ export default function Posts() {
 			{data.getAllPosts.map((post) => (
 				<Post key={post.id} post={post} />
 			))}
-			<button className="plusbutton"><NavLink className= "plusLink" to={"/NewPost"}><u className= "plusLink">+</u></NavLink></button>
+			<button className="plusbutton">
+				<NavLink className="plusLink" to={"/NewPost"}>
+					<u className="plusLink">+</u>
```
Comment (pennane) [inline packages/app/src/components/Posts.tsx:74]: da fak is `u` element? til

File: packages/app/src/components/Posts.tsx [hunk 3]
```diff
@@ -76,7 +69,11 @@ export default function Posts() {
 			{data.getAllPosts.map((post) => (
 				<Post key={post.id} post={post} />
 			))}
-			<button className="plusbutton"><NavLink className= "plusLink" to={"/NewPost"}><u className= "plusLink">+</u></NavLink></button>
+			<button className="plusbutton">
```
Comment (pennane) [inline packages/app/src/components/Posts.tsx:72]:
```suggestion
			<StyledButton>
```

File: packages/server/src/api/resolvers/baitResolver.ts [hunk 4]
```diff
@@ -1,28 +1,53 @@
...
+	Mutation: {
+		createBait: async (
+			_parent: any,
+			args: any,
+			context: TContext,
+			_info: any,
+		) => {
+			if (!context.user) {
+				throw new Error("Log in!");
+			}
```
Comment (pennane) [inline packages/server/src/api/resolvers/baitResolver.ts:41]: super secure bullet proof

File: packages/app/src/components/Posts.tsx (hunk 1 above)
Comment (perttuvaarala) [reply packages/app/src/components/Posts.tsx:68]: no working

**Review by perttuvaarala — APPROVED** [review]
looking good

**Pull request merged**